VirtualH89/Src/GUIglut.cpp

```diff
@@ -1,3 +1,4 @@
+#if defined(__GUIglut__)
```

Comment:
Instead of having 2 defines, if we make the checks for __GUIglut__,  `#ifndef __GUIwx__`, then it'll still build a valid program no matter what is defined. With the 2 defines, if either both are defined or neither are defined, then it won't build a valid executable.